user_docs/en/userGuide.md

```diff
@@ -280,6 +280,76 @@ To open NVDA's menu, press `NVDA+n`.
+1. Press `tab` to the "Actions" button, which can be used to install the add-on, among other actions.
+1. Press `tab` to "Other Details", which lists details such as the publisher, version and homepage.
+1. To return to the list of add-ons, press `alt+a`, or `shift+tab` back to the "Available add-ons" list.
+
+#### Searching for add-ons {#SearchingForAddons}
+As well as browsing all available add-ons, it is possible to filter the add-ons shown.
+To search, press `alt+s` to jump to the "search" field and type the text to search for.
+The search field can contain information including the add-on ID, display name, publisher, author or description.
+The list changes while typing the search terms.
+Once done, press `tab` to go to the filtered list of add-ons and browse the results.
+
+#### Installing add-ons {#InstallingAddons}
+
+To install an add-on:
+1. With the focus on an add-on you would like to install, press `enter`.
+1. The actions menu opens with a list of actions including: "Install", "Home page", "License", "Source code", "Community reviews".
+1. To install the add-on, press `i` or `downArrow` to "install" and press `enter`.
+1. The focus returns to the add-on in the list and NVDA will read the details about the add-on.
+1. The "Status" information reported by NVDA changes from "Available" to "Downloading".
+1. Once the add-on has finished downloading, it will change to "Downloaded. Pending install".
+1. Repeat with any other add-ons you would like to install at the same time.
+1. Once finished, press `tab` until the focus is on the "Close" button, then press `enter`.
+1. The downloaded add-ons will start the installation process once the Add-on Store is closed.
+Add-ons may create dialogs during the installation process that you will need to respond to.
+1. When the add-ons have been installed, a dialog appears advising that changes were made, and you must restart NVDA for the add-on installation to complete.
+1. Press `enter` to restart NVDA.
+
+#### Managing installed add-ons {#ManagingInstalledAddons}
+Press `control+tab` to move between the tabs or sections of the Add-on Store.
+The tabs include: "Installed add-ons", "Updatable add-ons", "Available add-ons" and "Installed incompatible add-ons".
+Each of the tabs are set out similar to each other, as a list of add-ons, a panel for more details on the selected add-on, and the ability to perform actions for the add-on.
+The actions menu of installed add-ons includes "Disable" and "Remove" rather than "Install".
+Disabling an add-on stops NVDA from loading it, but leaves it installed.
+To re-enable a disabled add-on, activate "Enable" from the actions menu.
+After enabling, disabling, or removing add-ons, you will be prompted to restart NVDA when closing the Add-on Store.
+Note that in this dialog `escape` works the same as the close button.
+Any changes you had made in the dialog have already happened, but they don't take effect until NVDA restarts.

```

Comment:
I guess we should remove the sentence regarding the Escape key/close button. Or at least put it elsewhere to avoid separating two related sentences.

```suggestion
After enabling, disabling, or removing add-ons, you will be prompted to restart NVDA when closing the Add-on Store.
These changes will only take effect once NVDA is restarted.
```

Reply:
Perhaps swapping the original lines 343 and 344 rather than removing the note on escape?  I included it because the dialog does behave differently to many others where escape cancels changes.

Note also the proposed change to the wording of line 344 [in this comment](https://github.com/nvaccess/nvda/pull/16553#discussion_r1606398621) although that wouldn't change your point.

Reply:
OK to keep the note on `Escape`.
So:

```suggestion
After enabling, disabling, or removing add-ons, you will be prompted to restart NVDA when closing the Add-on Store.
These changes will only take effect once NVDA is restarted.
Note that in this dialog escape works the same as the close button.
```

Re your suggestion in https://github.com/nvaccess/nvda/pull/16553#discussion_r1606398621, I prefer the simpler wording that I have suggested. If the two related lines follow each other, there is no need to repeat the example of actions twice.